src/software/er_force_simulator_main.cpp

```diff
@@ -110,6 +110,9 @@ int main(int argc, char **argv)
         auto yellow_ssl_wrapper_output =
             ThreadedProtoUnixSender<SSLProto::SSL_WrapperPacket>(runtime_dir +
                                                                  YELLOW_SSL_WRAPPER_PATH);
+        auto common_ssl_wrapper_output =
+            ThreadedProtoUnixSender<SSLProto::SSL_WrapperPacket>(runtime_dir +
+                                                                 SSL_WRAPPER_PATH);
```

Comment:
Is this so the autoref receives vision?

Reply:
Yes we need to pass in Referee commands and Vision to Autoref